### PR TITLE
Update AEM dependencies to 6.5.10

### PR DESCRIPTION
**Pull request by kwin**

### CODE REVIEW

**Comment by kwin** [comment]
Testing with JDK17 still fails for some reason....

**Review by davidjgonzalez — CHANGES_REQUESTED** [review]
GActions are failing

**Comment by kwin** [comment]
> GActions are failing

Yes, I know, compare with https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/3081#issuecomment-1493399882. Any pointers appreciated.

**Comment by kwin** [comment]
@stefanseifert Any chance you have an idea why the `RemoteAssetDecoratorTest` fails with newer dependencies? It seems that the `RemoteAssetDecorator` being registered in `RemoteAssetDecoratorTest.setupFinish()` is never bound as `ResourceDecorator` in https://github.com/apache/sling-org-apache-sling-resourceresolver/blob/46f67c7cff13071d96cf58dde86730b6ffbb2639/src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverFactoryActivator.java#L609....

File: pom.xml [hunk 1]
```diff
@@ -104,7 +104,6 @@
                     <version>3.9.0</version>
                     <configuration>
                         <showDeprecation>true</showDeprecation>
-                        <failOnError>false</failOnError>
```
[the remark this answers is not in the document]
Comment (davidjgonzalez) [reply pom.xml:107]: hmmm - i do not recall :(

**Comment by kwin** [comment]
@davidjgonzalez Anything outstanding here?

**Comment by davidjgonzalez** [comment]
Nope! I must have missed it on last release! Merged

**Pull request merged**